moml/core/hierarchical_graph_coarsener.py

```
@@ -24,10 +24,11 @@
 from typing import Dict, List, Tuple, Optional, Union
 import logging
 import numpy as np
+from pathlib import Path
```

Ruff (0.11.9)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)